pandas/io/stata.py

```diff
@@ -2135,7 +2165,6 @@ def value_labels(self) -> dict[str, dict[int, str]]:
 
 
 @set_module("pandas")
-@Appender(_read_stata_doc)
```

Comment:
Can `_read_stata_doc` (and any associated variables) be removed after this change?